fudge-ratio/src/main/webapp/index/index.html

```diff
@@ -23,12 +23,12 @@ <h2>Calculate yours below!</h2>
                 </div>
                 <div class="startIcon">
                     <div id="text">
-                        <a href="calculator.html" class="fill-div">Get Started!</a>
+                        <a href="/calculator" class="fill-div">Get Started!</a>
```

Comment:
I think its a good practice to make all links to visit another pages absolute and delete the extension, giving you more flexibility